Cargo.toml

```diff
@@ -10,14 +10,15 @@ keywords = ["lru", "concurrent", "cache"]
 categories = ["caching", "concurrency", "data-structures"]
 readme = "README.md"
 exclude = ["fuzz"]
+rust-version = "1.63"
```

Comment:
Just curious. How accurate is this? Could it be even lower?

Reply:
I determined the MSRV using [`cargo-msrv`](https://github.com/foresterre/cargo-msrv) on a x64 Linux box, so it's spot on! :wink:

- With the `parking_lot` feature disabled, `cargo msrv -- cargo test` returns that the MSRV is 1.63.
- With the `parking_lot` feature enabled, the command above returns that the MSRV is 1.61. This is because `parking_lot::RwLock::new` has always been `const`, unlike `std:.sync::RwLock::new`.

Therefore, with a `Cargo.lock` pointing to the latest semver-compatible versions of dependencies, MSRV can't go lower than 1.63 without sacrificing feature choices.

Reply:
This https://github.com/foresterre/cargo-msrv tool is really nice!